docs/usage.md

```diff
@@ -105,20 +105,27 @@ The parameter `--map-base-url` maps external schema references to local folders.
 
 When using our docker image that we regularly update, you don't need to install Node.js or Npm, even though the generator is written with it since the Docker image has the generator installed.
 
-Install [Docker](https://docs.docker.com/get-docker/) first, then use docker to pull and run the image using the following command:
+Install [Docker](https://docs.docker.com/get-docker/) first, then use docker to build the image using the following command :
+``` 
+docker build -t asyncapi/cli:latest . 
```

Comment:
sorry but this was relevant only inside CLI docs, in Generator it will be confusing as there is not docker file for AsyncAPI CLI in the generator.

so please stick to `Install [Docker](https://docs.docker.com/get-docker/) first, then use docker to pull and run the image using the following command:`

sorry I should have been more clear in the issue